packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -11,16 +11,6 @@
  */
 
 .root {
-  color: $text-body;
-  font-family: $body-font-family;
-  font-size: 1rem;
-  font-style: normal;
-  font-weight: 400;
-  letter-spacing: 0;
-  line-height: $base-line-height;
-  font-feature-settings: "lnum", "pnum";
-  text-decoration-skip-ink: all;
-
```

Comment:
just validating that all these styles are provided via box now and will inherit as well?

Reply:
That's correct, they will.
https://github.com/okta/odyssey/blob/ab/text-api-refactor/packages/odyssey-react/src/components/Box/_box-root.scss#L13